.travis.yml

```diff
@@ -9,473 +9,473 @@ cache:
+    # of ansible/ansible:
+
+    - <<: *py-37
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        ANSIBLE_VERSION: devel
+        TESTS_TYPE: unit
+    - <<: *py-37
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_1
+        ANSIBLE_VERSION: devel
+    - <<: *py-37
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_2
+        ANSIBLE_VERSION: devel
+    - <<: *py-37
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_3
+        ANSIBLE_VERSION: devel
+
+    - <<: *py-36
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        ANSIBLE_VERSION: devel
+        TESTS_TYPE: unit
+    - <<: *py-36
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_1
+        ANSIBLE_VERSION: devel
+    - <<: *py-36
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_2
+        ANSIBLE_VERSION: devel
+    - <<: *py-36
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_3
+        ANSIBLE_VERSION: devel
+
+    - <<: *py-27
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        ANSIBLE_VERSION: devel
+        TESTS_TYPE: unit
+    - <<: *py-27
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_1
+        ANSIBLE_VERSION: devel
+    - <<: *py-27
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_1
+        ANSIBLE_VERSION: devel
+    - <<: *py-27
+      <<: *if-cron-or-manual-run-or-tagged
+      env:
+        <<: *env-functional-shard_3
+        ANSIBLE_VERSION: devel
+
+    - &deploy-job
+      <<: *py-37
+      <<: *reset-prerequisites
+      stage: Deploy
+      name: Publishing current Git tagged version of dist to PyPI
+      if: repo == "ansible/molecule" AND tag IS present
+      env:
+        TOXENV: metadata-validation,build-dists
+      before_deploy:
+        - echo > setup.py
+      deploy: &deploy-step
+        provider: pypi
+        user: ansible-molecule
+        password:
+          secure: >
+            HPBkDwncLYLEzxirPfNB88GYFWZ/wA1jMgCd4wT7SgKQlIyRSCT6KXMfoOUVMH/uwaLggoURaGtmD/qnrfXj+bG15nBCCOaIZdXzODln/PwDFAYT8ZspzRPzQOfncdk4WTsVbwpiGgpdm+TxGGBz8yedvVXiTwmwLCGC0FsAE8Wp8krNH1Kwqp3OaZeePakIbEj0UXgCTnAol3ZgVWWy+6bfDBb/aLiGXjsAIb7sY6HzwvQUr43xFO7tReaGO23mGwgIy/tNstXarwxezlw6FlGe5KJGvE/a4yAPuWg9kuJt5MqwbCJzhP38SeH4Gc6x0o6jPT/+NbvLV1ck1Qbz5gCYmXlOzucDP+P5t8E3g+zJbNbCR00k1BpG3MIEzrHeyp/hCdDYnGB3TVnlj+L4Y/yoUdCq9FJI8xHLbj/fMe5vxBMMsvDbt2JJReavi3WB2pv2nHtivkKj/Kow4FRm+Kp3WH4x5NlZDjV5gsUbwPcN1kIpb09sPZE2wYugvCTOiExt7mdd0JMMcgxuR1jGgGxqRARQeqjz14VVNsrVIIrMh8JX8u7Rl9sqmU4UuLGTxnq2cwyBmYzJ0gmDgN3TvQ96n9D2sP0YQj35v7RY5vsdYsfLucLdTfTajP9OL4Bw+ogXMJrArIq+j+7uJyTFqNdAN+Y6nX7WV0W6rp8aA1I=
+        distributions: dists
+        skip-cleanup: true
+        "on":
```

Comment:
This is broken.

Reply:
That is a valid yaml syntax trick for making yaml syntax compliant. Mainly unquoted strings like "on", "true"... are considered booleans when loaded. Adding quotes assures that remain strings, like it is expected by travis. I totally agreet hat is ugly, but is the price we pay for and old travis decision of not using a linter when they wrote their yaml configs.

Reply:
indentation.

Reply:
Resolving as it turned out that `?w=1` is buggy and lies to viewers.

cc @gundalow we shouldn't rely on that view.